src/main/resources/peer-review-transform.xsl

```diff
@@ -106,17 +105,23 @@
     <tr class="peer-review-accordion-item">
       <xsl:attribute name="data-doi"><xsl:value-of select="current()/@id" /></xsl:attribute>
       <td class="letter">
-        <div class="acceptance-letter" itemscope="" itemtype="http://schema.org/Review">
```

Comment:
@samV-plos I wanted you to know that I removed all of the `itemprop` and related attributes that were in the original prototype.  It really needs a ticket if we want to implement it with real data.

Reply:
Ok, that's fine.  Thanks for the heads up!